ethereum/spec/src/main/java/tech/pegasys/teku/spec/config/SpecConfigBuilder.java

```diff
@@ -109,6 +109,7 @@ public class SpecConfigBuilder {
 
   // Bellatrix
   private Optional<BellatrixBuilder> bellatrixBuilder = Optional.empty();
+  private boolean firstDescendentAsChainHeadEnabled = true;
```

Comment:
hmm, this probably shouldn't wind up in SpecConfig if we can avoid it. The progressive balances feature toggle wound up in there out of necessity but we shouldn't need this in the milestone specific code - just `ForkChoice` so could just pass it there I think?

Reply:
it did seem messy, just seemed how other eth2 things got passed, will refactor